src/caret_analyze/plot/visualize_lib/bokeh.py

```diff
@@ -340,8 +345,13 @@ def _get_data_dict(
         target_object: TimeSeriesTypes
     ) -> Dict[str, Any]:
         data_dict: Dict[str, Any] = {}
+
         for k in self._get_source_keys(target_object):
             try:
+                if k == 'legend_label':
+                    label = self._legend.get_label(target_object)
+                    data_dict['legend_label'] = [f'legend_label = {label}']
+                    continue
```

Comment:
```suggestion
```

Reply:
@hsgwa 
How about moving this process into the `_get_description` as follows?

```
def _get_description(
    self,
    key: str,
    target_object: TimeSeriesTypes
) -> str:
    if key == 'callback_param':
        if isinstance(target_object, TimerCallback):
            description = f'period_ns = {target_object.period_ns}'
        elif isinstance(target_object, SubscriptionCallback):
            description = f'subscribe_topic_name = {target_object.subscribe_topic_name}'
    elif key == 'legend_label':
        label = self._legend.get_label(target_object)
        description = f'legend_label = {label}'
    else:
        raise NotImplementedError()

    return description
```

Reply:
addressed in :[3bde03c](https://github.com/tier4/CARET_analyze/pull/222/commits/3bde03c234442de76964d4d7187775940ed8277b)